web/modules/custom/dpl_patron_page/src/Plugin/Block/PatronPageBlock.php

```diff
@@ -103,7 +97,7 @@ public function build() {
       'pause-reservation-start-date-config' => $dateConfig,
       'pincode-length-max-config' => $patron_page_settings->get('pincode_length_max'),
       'pincode-length-min-config' => $patron_page_settings->get('pincode_length_min'),
-      'text-notifications-enabled-config' => (int) $this->textNotificationsEnabled(),
+      'text-notifications-enabled-config' => (int) $this->reservationSettings->smsNotificationsIsEnabled(),
```

Comment:
This will not work you have to loadConfig first into variable as with the `patron_page_settings`